checks/worker/binary_artifact.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package worker
```

Comment:
"worker" doesn't seem like the most suitable name to me. It's overloaded with the cron job "worker". 

I also think it's better to just merge "raw" and "worker". Having everything in one file like C headers (which is a convention mostly out of necessity in C) doesn't seem like enough of a benefit to me to justify the additional complexity.

Reply:
+1. I would even go a step further - put all the `raw` and `worker ` logic inside `evaluation` package.

Reply:
`worker` just retrieves the results, it cannot be inside `evaluation`. I'm going to simplify: I'm going to remove worker entirely and put the code back into `checks/binary_artifact.go`. Since the long-term intention is for the checks to return only the raw results, it will be done in `checks/binary_artifacts.go` anyway.
I'll merge the raw's structure definition into the `checks/binary_artifacts.go`

Last question: should `evaluation` live under `check` or not? Long-term, evaluation of the policy/score will be done outside the call to `pkg.RunScorecards`. So maybe I could move it under `evaluation/score` under the root folder. wdut?

Reply:
SGTM. Also ok if you want to move to `evaluation` root package in a separate PR.

Reply:
thanks both. I'll move the validation to a differnent package in a sub-sequent PR.